ubuntudesign/maas.io
# Remove all mention of local docs from site

Now that docs are stored over on docs.ubuntu.com/maas, remove them from this site completely:

- Remove CSS and JS
- Remove building instructions from README
- Remove templates, includes and directories for building docs
- Remove the `import-docs` script and the `make docs` target

QA
--

```
npm install --legacy-bundling || npm install
make sass
make setup develop
```

Check the site still hangs together and looks fine.
